test/com/google/javascript/jscomp/ModuleRewriteIntegrationTest.java

```diff
@@ -262,11 +261,127 @@ public void testCrossModuleSubclass6() {
         });
   }
 
+  @Test
+  public void testCommonJSImportsESModule() {
```

Comment:
Also fwiw I'm pretty sure I had nothing to do with this one since I didn't touch the CJS pass :)

I'll take a look at the others, but can you handle this one?